main.ppk

```diff
@@ -0,0 +1,26 @@
+PuTTY-User-Key-File-2: ssh-rsa
+Encryption: none
+Comment: main
+Public-Lines: 6
+AAAAB3NzaC1yc2EAAAADAQABAAABAQCHiMGCfZ26Z4cn5gQqJ3g8qUwq1xmSBL4E
+9De16hiGZwXWPc9474eVYvOJsnL3Tfr7NY4x1H0s/dhJz7qLnnn6yPt/Fk6uhOFq
+qk8ejSBKAkPBCXFgCAqV5HaQ6/U/QRRJhjrIT/ud9ZjLuCYkeSwYb3XGXitHOmWM
+aAUxi0A5XC/08BOG9RM7YmXrK0HWaeDtrFkLx69wWf+dY0YwRPqH+coe/2p0to/2
+z46+vJEopaqhYpuE1gFUURseSZ8AvlzRZntOl8iwzHQ3Pxx97PqGT6ziTBoTVwx7
+junjo+juWS1Id+ql7VrM0kb8i/1BjY/GO7j0Rv7iqgU20X/BgxLl
+Private-Lines: 14
+AAABAERfO+d9QSV49pitE6rEUIahjTgLXO3pkIZlQPl3R7n9nNcwRUqn40j8Kwlq
+yU5yu0shAEOpAyDASe0qatvmLXjTDPRWp4iopiKuGc26ufA75jwqbFRamiR1pU6E
+iirHxNk2as8EclWEQPl8Q1mg+xq0l8BjXwfGsNW83+nXaqXEWhk+JOYWL+VjW0nr
+4V0tD5mfg9b9VzuJgKtDCbpOeAZeMEeYcj2cQH5TJLxXxUfx2hstSzGocInHMxsX
+FgdHGVa1ul0qPycXTGp6p+3CEySSjKLcQGRzMNLAy/b1LGE9OlCSzO/YnddwjLR8
+jATRJeUO3rx08UDD/J2SDZOTWR0AAACBANyM7Fm+tBne5HiSAh+YWTYbO3QXO98b
+qCkrI4PTwkCnE5/AYlsB6idMYz1GtjRmmzAPAYjSdyaZa/EDxEWytua4QRyXbp5m
+UjvGlFYPNtQT2cDMMlvRlhYYuO+bLhaOzhJn9zmoO9WExDc5wmUn2FaK6oT82mXu
+d1O4ttHTRdyDAAAAgQCdUaJ4IBXAVeCQNc265oI9S8jmk9LbQ+jKPqZ1TDmclEsY
+Cv8DjelcL9WoOHjTftAaGSfDJJIs3HTdZYdErIblKUxDiZZD6yONz7EdG3qqzjCQ
+bxRKO+8eL9d3cHX7VPYc0nKiJl9MHsm93DDwhpGzH+J32114KddrqNeDYl+GdwAA
+AIBfqXmrwVwLODYMLX2yl+wZZiFQ++ql/aY7EBSbzHXGPDbKCIbfAW25lBsE2HiY
+J/JZY5JzoE30zW2BN/GwQP9TPqaKEH8P6TSEiiO2bLsfquL5/nhmMlhPSh/DiN8I
+v9PHmjAWULQTSR+iACogBt8K3P5xyPLgcUn2RC8PZf+glw==
+Private-MAC: 13d14f02516beb4eafffe394ed76b69a582a404d
```

Comment:
could you also delete this key as it is not a good security practice to have it here.